deploy/scaleway-webhook/templates/deployment.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  replicas: {{ .Values.replicaCount }}
```

Comment:
Any reason to remove this param?

Reply:
replicaCount wasn't define and I'm not sure we can use multiple replica here

Reply:
actually there's no issue with multiple replicas, I'll edit